### ISSUE TITLE
Athletics Widget Format

### ISSUE THREAD

**Issue by Lev1ty** [the-reckoner/reckoner-site#3]
Athletics widget on splash page does not conform with color scheme, format of rest of page
Basically does not look pretty

Look into css? Good chance to experience css

**Comment by AkilMiki** [comment]
I agree the padding is nonexistent, I'll look into it

**Comment by Lev1ty** [comment]
@AkilMiki and I attempted to find the css for the table widget. I first thought it was a custom css module, but turned out it was a widget. Then, we looked into the css customization of divi modules through the following navigation:
![screenshot from 2018-02-17 20-59-17](https://user-images.githubusercontent.com/15148828/36347467-c8f694b0-1425-11e8-8901-f17d32ec5f89.png)
![screenshot from 2018-02-17 21-02-26](https://user-images.githubusercontent.com/15148828/36347470-ea0b46be-1425-11e8-931f-214ac0bdf30e.png)
@AkilMiki indicated that the divi modules were ineffective. I verified by testing extraneous parameters in the above image. Upon realizing again that the table was indeed a widget, I investigated the table widget: *Tables by Supsystic* and found the css native to the table. This was verified by inspect element of the reckoner home page.
![screenshot from 2018-02-17 21-05-30](https://user-images.githubusercontent.com/15148828/36347514-758dc5ea-1426-11e8-8b32-0b6d7c5a1f07.png)
@AkilMiki then took charge of fixing the css of interest.
